JS/Lesson27/goksel/script.js

```diff
@@ -20,79 +20,139 @@ HTTP status codes are three-digit numbers that the server sends in response to a
 
 // Mocked - it behaves like if the request was processed even if it is not
 
-const URL = 'https://jsonplaceholder.typicode.com/posts';
+// const URL = 'https://jsonplaceholder.typicode.com/posts/1';
 
 function getPosts() {
 	fetch('https://jsonplaceholder.typicode.com/posts')
 		.then((response) => response.json())
 		.then((data) =>
+
 			data.map((item) => {
 				//  post-container
 				const post = document.createElement('div');
 				post.classList.add('post');
+
 				const postTitle = document.createElement('h2');
-				postTitle.innerText = item.title;
-				post.classList.add('post-title');
+				postTitle.classList.add('post-title');
+				postTitle.innerText = item.title.toUpperCase();
+
 
 				const postBody = document.createElement('p');
 				postBody.innerText = item.body;
+
 				post.classList.add('post-body');
 
 				const postsContainer = document.getElementById('post-container');
 				// TODO: add <a> element instead of <button>, href will be './editPost/edit-post.html?postId=${postId}';
-				const editPostButton = document.createElement('button');
+
+				const editPostButton = document.createElement('a');
+				editPostButton.href = `./editPost/edit-post.html?postId=${item.id}`;
+				editPostButton.classList.add("edit-btn");
 				editPostButton.textContent = 'Edit Post';
-				editPostButton.onclick = () => updatePost(item.id);
 
-				post.appendChild(postTitle);
+
+				const deletePostBtn = document.createElement("button");
+				deletePostBtn.classList.add("delete-btn");
+				deletePostBtn.textContent = "Delete Post"
+				deletePostBtn.onclick = () => {
+					post.style.display = "none";
+				}
+
+
+				const header = document.createElement("div");
+				header.classList.add("header");
+				const btn = document.createElement("div");
```

Comment:
I see that you're creating a div element but due to the name "btn" my initial assumption was that it's a "button" element. The name is confusing in this situation. How about "btn-container" or "post-actions"?